src/PolicyInsights/PolicyInsights/Az.PolicyInsights.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.4.1'
+ModuleVersion = '1.5.0'
 
```

Comment:
Please don't update the version and release notes in this file manully, our tool will update them.

Reply:
And please update the changelog.md